test/fixtures_public.js

```diff
@@ -7,6 +7,7 @@
 // 05059996666: registered user - mother that receives own messages, linked to 05059997777
 // 05059997777: registered user - family member that receives household messages for 05059996666
 // 05059998888: number being changed to
+// 05059999999: number used to test sms app; does unsubscribe
```

Comment:
doing an opt-out require a GET with a subscription url so i added a new number for this reason... i guess I could add a subscription-GET-request to ..22222... ?